lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -1000,7 +1006,9 @@ def accessible_operations(user_or_attribute, object_or_attribute, operation_id =
         prms = { type: PolicyMachineStorageAdapter::ActiveRecord::Operation.to_s }
         prms.merge!(unique_identifier: operation_id) if operation_id
 
-        Assignment.descendants_of(associations.map(&:operation_set)).where(prms)
+        operation_sets = OperationSet.where(id: associations.select(:operation_set_id))
```

Comment:
Looks like you should be able to use `pluck` instead of `select` here.

Reply:
See comment above - `pluck` will actually load the IDs, while we just want the _relation_ used to build the query, similar to above.